src/util/detectOS.ts

```diff
@@ -9,12 +9,27 @@ export enum UserOS {
 
 export function detectOS(): UserOS {
   let OS = UserOS.UNKNOWN;
-  if (typeof navigator !== `undefined`) {
-    if (navigator.appVersion.indexOf('Win') !== -1) OS = UserOS.WIN;
-    if (navigator.appVersion.indexOf('Mac') !== -1) OS = UserOS.MAC;
-    if (navigator.appVersion.indexOf('X11') !== -1) OS = UserOS.UNIX;
-    if (navigator.appVersion.indexOf('Linux') !== -1) OS = UserOS.LINUX;
-    if (navigator.appVersion.indexOf('Mobi') !== -1) OS = UserOS.MOBILE;
+  if (typeof window !== 'undefined') {
+    switch (typeof navigator !== `undefined`) {
```

Comment:
I assume backtick quotes are not needed here

Reply:
fixed...